app/src/ui/publish-repository/publish-repository.tsx

```diff
@@ -64,7 +65,8 @@ export class PublishRepository extends React.Component<
 
   private async fetchOrgs(account: Account) {
     const api = API.fromAccount(account)
-    const orgs = await api.fetchOrgs()
+    const orgs = (await api.fetchOrgs()) as Array<IAPIUser>
+    orgs.sort((a, b) => compare(a.login, b.login))
```

Comment:
Something to call out: this `compare` seems to be case-sensitive. I have no strong feels either way about whether to do this or make it case-insensitive, but this is currently better than what it was:

<img width="90" src="https://user-images.githubusercontent.com/359239/34786661-2e8259a6-f60b-11e7-834b-51f3abfdaca1.png">